Source/ASTextNode2.mm

```diff
@@ -318,6 +318,19 @@ - (UIEdgeInsets)textContainerInset
   return _textContainer.insets;
 }
 
+- (void)setTextContainerLinePositionModifier:(id<ASTextLinePositionModifier>)modifier
+{
+  ASLockScopeSelf();
+  if (_textContainer.linePositionModifier != modifier) {
+    _textContainer.linePositionModifier = modifier;
```

Comment:
```suggestion
ASLockedSelfCompareAssignObjects(_textContainer.linePositionModifier, modifier);
```

Reply:
I think delete the `ASLockScopeSelf()` as well if you take accept this suggestion (as it's built into the macro)

Reply:
Yeah not need for `ASLockScopeSelf`, see updated version.